handbook/company/leadership.md

```diff
@@ -374,13 +374,15 @@ Once the new team member replies and accepts their offer in writing, 🔦 Head o
      ```
 2. **Ask hiring manager to send rejections:** Post to the `hiring-xxxxx-yyyy` Slack channel to let folks know the offer was accepted, and at-mention the _hiring manager_ to ask them to communicate with [all other interviewees](https://fleetdm.com/handbook/company#empathy) who are still in the running and [let them know that we chose a different person](https://fleetdm.com/handbook/company/leadership#candidate-correspondence-email-templates).
    >_**Note:** Send rejection emails quickly, within 1 business day.  It only gets harder if you wait._
-3. **Remove open position:** The hiring manager removes the newly-filled position from the fleetdm.com website by [making a pull request](https://fleetdm.com/handbook/company/communications#making-a-pull-request) to delete it from the [open-positions.yml](https://github.com/fleetdm/fleet/blob/main/handbook/company/open-positions.yml) file.
-4. **Close Slack channel:** Then archive and close the channel.
+3. **Remove open position:** Ensure the hiring manager removes the newly-filled position from the fleetdm.com website by [making a pull request](https://fleetdm.com/handbook/company/communications#making-a-pull-request) to delete it from the [open-positions.yml](https://github.com/fleetdm/fleet/blob/main/handbook/company/open-positions.yml) file.
+4. **Create 30-60-90 day plan:** Ensure the hiring manager creates a 30-60-90 day plan outlining key role objectives. The plan is reviewed weekly in 1:1 meetings during the first three months of employment, ensuring continuous support and alignment with company goals. To create the 30-60-90 day plan, hiring manager will:
+    - Create a copy of the [30-60-90 day plan template](https://docs.google.com/document/d/1EztmPBuMFXbVoy4ZToXcxasNO38ooOh8Gh5hPXFvJhI/copy) and rename the copied file using the naming convention `[start date] - 30-60-90 day plan - [teammate full name]` and move it to the [30-60-90 day plan folder](https://drive.google.com/drive/u/0/folders/1QWiAbgBFuuofT_3M8oIoBsbEBmubQAj7) in Google Drive.
+    - Follow the prompts in the template to fill out the 30-60-90 day plan for the new teammate before they start.
+5. **Close Slack channel:** Then archive and close the channel.
```

Comment:
Is this the hiring manger or BizOps? I know this pre-dates this change but I think we should be explicit here. From what I've seen it's been BizOps (and I think that's what we're saying here) but it could just as easily be the hiring manager. thoughts @ireedy and @JoStableford?

I can also get Mike's feedback when he's back in office.

Reply:
This is the hiring manager. BizOps does not have the context of what success in the individual roles look like. BizOps is [creating the structure and ensuring compliance](https://github.com/fleetdm/confidential/issues/7410) with the process that [every new starter has a 30-60-90 day plan](https://docs.google.com/spreadsheets/d/1Hso0LxqwrRVINCyW_n436bNHmoqhoLhC8bcbvLPOs9A/edit?gid=1846478041#gid=1846478041&range=B380).

Reply:
Oh sorry, I meant just this bit: `5. **Close Slack channel:** Then archive and close the channel.`

~I didn't want to add yet another step on the BizOps plate related to hiring if it didn't belong, but also going off of what I've seen in practice.~ I also just refreshed myself with this piece:

`Once the new team member replies and accepts their offer in writing, 🔦 Head of Business Operations follows these steps:`

It's covered! Sorry for the wild goose chase. 🪿🏃‍♂️   LGTM!!!